src/statespace/dart/MetaSkeletonStateSpaceSaver.cpp

```diff
@@ -11,6 +14,7 @@ MetaSkeletonStateSpaceSaver::MetaSkeletonStateSpaceSaver(
 {
 }
```

Comment:
Nit: Please add `// Do nothing` to indicate this function is intentionally empty.